__tests__/app.test.js

```diff
@@ -88,13 +99,40 @@ describe('app', () => {
     });
   });
 
-  describe('/api/badPath', () => {
-    it('404: GET responds with not found error, if incorrect path given', () => {
+  describe('/api/articles/:article_id', () => {
+    it('200: GET responds with object of single article', () => {
       return request(app)
-        .get('/api/badPath')
+        .get('/api/articles/3')
+        .expect(200)
+        .then(({ body: { article } }) => {
+          expect(article).toEqual(
```

Comment:
In order for this to be more flexible further down the line (say if we wish to add another column) we would need to look to use a more flexible matcher for our tests that toEqual - for instance objectContaining or toMatchObject 🙂